webextensions/manifest/background.json

```
@@ -146,6 +146,27 @@
               "safari_ios": "mirror"
             }
           }
+        },
+        "type": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": true,
+                "notes": "When set to <code>module </code>, includes service workers as ES modules."
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "112",
+                "notes": "When set to <code>module </code>, includes background scripts as ES modules."
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
```

Safari supports this in 16.4 per https://github.com/w3c/webextensions/issues/289#issuecomment-1433617753
```suggestion
                "version_added": "16.4"
```